### PR TITLE
feat: add JobIdentity field and add get scheduler job by identity api

### PR DESCRIPTION
**Pull request by gavin1ee**

### CODE REVIEW

File: src/Contrib/StackSdks/Masa.Contrib.StackSdks.Scheduler/_Imports.cs [hunk 1]
```diff
@@ -18,3 +18,4 @@
 global using Microsoft.Extensions.DependencyInjection;
 global using Microsoft.Extensions.Logging;
 global using static Masa.Contrib.StackSdks.Scheduler.Constants;
+global using Microsoft.AspNetCore.Builder;
```
Comment (zhenlei520) [inline src/Contrib/StackSdks/Masa.Contrib.StackSdks.Scheduler/_Imports.cs:21]: global using sort

**Review by zhenlei520 — CHANGES_REQUESTED** [review]

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        public int FailedRetryInterval { get; set; }
+
+        public int FailedRetryCount { get; set; }
+
+        public string Description { get; set; } = string.Empty;
+
+        public bool Enabled { get; set; }
+
+        public Guid BelongTeamId { get; set; }
```
[the remark this answers is not in the document]
Comment (gavin1ee) [reply src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs:42]: It was named by domain model.  if necessary to rename it, I need to rename doamin model.  if not necessary to rename it, I will leave it alone.

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        public string BelongProjectIdentity { get; set; } = string.Empty;
```
[the remark this answers is not in the document]
Comment (gavin1ee) [reply src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs:44]: It was named by domain model.  if necessary to rename it, I need to rename doamin model.  if not necessary to rename it, I will leave it alone.

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+        public string CronExpression { get; set; } = string.Empty;
+
+        public string JobIdentity { get; set; } = string.Empty;
+
+        public JobTypes JobType { get; set; }
+
+        public string SpecifiedWorkerHost { get; set; } = string.Empty;
+
+        public ScheduleExpiredStrategyTypes ScheduleExpiredStrategy { get; set; }
```
[the remark this answers is not in the document]
Comment (gavin1ee) [reply src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs:26]: It was named by domain model.  if necessary to rename it, I need to rename doamin model.  if not necessary to rename it, I will leave it alone.

File: src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+        public ScheduleBlockStrategyTypes ScheduleBlockStrategy { get; set; }
```
[the remark this answers is not in the document]
Comment (gavin1ee) [reply src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdks.Scheduler/Model/SchedulerJobModel.cs:28]: It was named by domain model.  if necessary to rename it, I need to rename doamin model.  if not necessary to rename it, I will leave it alone.

**Review by zhenlei520 — APPROVED** [review]
@doddgu LGTM

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**